usr/local/share/vulture-utils/upgrade-os.sh

```diff
@@ -74,11 +74,8 @@ update_system() {
             _options="${_options} -v $system_version -U"
         fi
         if [ $snapshot_system -gt 0 ]; then
-            /sbin/bectl create "${_snap_name}" || finalize 1 "Could not create a new Boot Environment!"
             clean_old_BEs "$keep_previous_snap"
```

Comment:
```suggestion
```
This should not be done here unconditionaly

Reply:
Done